tests/installation/enable_y2_firstboot.pm

```diff
@@ -16,11 +16,16 @@ use strict;
 use warnings;
 use testapi;
 use utils qw(zypper_call clear_console);
+use scheduler 'get_test_suite_data';
 
 sub run {
+    my $test_data = get_test_suite_data();
     select_console 'root-console';
     zypper_call "in yast2-firstboot";
     assert_script_run 'touch /var/lib/YaST2/reconfig_system';
+    if ($test_data->{custom_control_file}) {    # Use default file from package if not custom file was specified
```

Comment:
my recommendation is that the comment should be on top, not on the right of some language keyword like `{`.